string processor_name    # name of soft-processor to soft-reset, 
                         # certain devices may have more than one soft-processor
---
bool success             # true if processor was successfully reset
string error_msg         # descriptive error message if call was not successful
